aplib.net-demo/Assets/Scripts/Inventory/Inventory.cs

```diff
@@ -66,7 +81,9 @@ public void ActivateItem()
             _itemList.Peek().UseItem();
 
             if (_itemList.Peek().uses == 0)
+            {
                 _ = _itemList.Dequeue();
+            }
```

Comment:
Volgens mij moet de styling een van deze twee zijn.
```suggestion
            if (_itemList.Peek().uses == 0)
                _ = _itemList.Dequeue();
```
```suggestion
            if (_itemList.Peek().uses == 0) _ = _itemList.Dequeue();
```